lib/IDE/CodeCompletion.cpp

```diff
@@ -4845,6 +4845,11 @@ static void addDeclKeywords(CodeCompletionResultSink &Sink) {
       // Treat keywords that could be the start of a pattern specially.
       return;
     }
+    // FIXME: __consuming should not appear in CodeCompletion until it is
+    // finalized in a language proposal.
+    if (Name == "__consuming")
+      return;
```

Comment:
We should form the DeclAttributeKind an use `DeclAttribute::isUserInaccessible()` instead of handling __consuming explicitly.  I thought we already did that, but it looks like we only handle it on attribute completion right now.